tests/components/sonos/test_init.py

```diff
@@ -322,16 +326,19 @@ async def test_async_poll_manual_hosts_5(
         # Speed up manual discovery interval so second iteration runs sooner
         mock_discovery_interval.total_seconds = Mock(side_effect=[0.5, 60])
 
-        await _setup_hass(hass)
-
-        assert "media_player.bedroom" in entity_registry.entities
-        assert "media_player.living_room" in entity_registry.entities
-
         with caplog.at_level(logging.DEBUG):
             caplog.clear()
-            await speaker_1_activity.event.wait()
-            await speaker_2_activity.event.wait()
+
+            await _setup_hass(hass)
+
+            assert "media_player.bedroom" in entity_registry.entities
+            assert "media_player.living_room" in entity_registry.entities
+
+            async_fire_time_changed(hass, dt_util.utcnow() + timedelta(seconds=0.5))
```

Comment:
Should we use the freezer here?

Reply:
That is what I thought initially. But it seems the calls happen, but the logging starts too late (see PR descr).